app/src/main/java/de/westnordost/streetcomplete/overlays/address/AddressOverlayForm.kt

```diff
@@ -50,11 +50,13 @@ class AddressOverlayForm : AbstractOverlayForm() {
     private var streetOrPlaceName: StreetOrPlaceName? = null
 
     private var isShowingHouseName: Boolean = false
-    private var isShowingPlaceName = false
+    private var isShowingPlaceName: Boolean = false
+    private var isShowingBlockNumber: Boolean = false
```

Comment:
Probably needs to be saved in `onSaveInstanceState` like the other two. Also, this is never set to `true`, right?

Reply:
> Probably needs to be saved in onSaveInstanceState

Probably. Though maybe it could also be read from `numberView.findViewById<EditText?>(R.id.blockNumberInput) != null`?

> Also, this is never set to true, right?

Set to true if there is a block number input in `isShowingBlockNumber = numberView.findViewById<EditText?>(R.id.blockNumberInput) != null`